src/metabase/models/serialization/util.clj

```diff
@@ -405,6 +404,31 @@
        (map mbql-fully-qualified-names->ids)
        (map #(m/update-existing % :card_id import-fk 'Card))))
 
+(defn export-parameters
+  "Given the :parameter field of a `Card` or `Dashboard`, as a vector of maps, converts
+  it to a portable form with the CardIds/FieldIds replaced with `[db schema table field]` references."
+  [parameters]
+  (map ids->fully-qualified-names parameters))
+
+(defn import-parameters
+  "Given the :parameter field as exported by serialization convert its field references
+  (`[db schema table field]`) back into raw IDs."
+  [parameters]
+  (->> parameters
```

Comment:
This seems more legible as `(for [param parameters] (-> param mbql-fully.qualified-names->ids (m/update-existing-in ...)))`.